django/db/backends/base/creation.py

```diff
@@ -268,6 +273,17 @@ def destroy_test_db(self, old_database_name=None, verbosity=1, keepdb=False, num
             settings.DATABASES[self.connection.alias]["NAME"] = old_database_name
             self.connection.settings_dict["NAME"] = old_database_name
 
+    def _restore_initial_data_migrations(self):
+        # First, clean the database
+        call_command('flush', verbosity=0, interactive=False,
```

Comment:
What does the flush do? I would expect the tests to leave the database in a clean state already.